src/routes/webhooks.ts

```diff
@@ -29,16 +26,23 @@ export default async function routes(fastify: FastifyInstance) {
       }
 
       switch (event.type) {
+        case 'charge.failed':
+        case 'charge.refunded':
+        case 'charge.succeeded': {
+          const charge = event.data.object as Stripe.Charge
+          await upsertCharge(charge)
+          break
+        }
         case 'customer.created':
         case 'customer.updated': {
-          const customer = event.data.object as Customer.Customer
+          const customer = event.data.object as Stripe.Customer
```

Comment:
This doesn't need to be Stripe.Customer.Customer right?

Reply:
No need. Previously on the import we do `import Customer from 'stripe'` so it need to be `Customer.Customer`

rename the import to Stripe namespace makes it clearer.